assets/js/phoenix_live_view.js

```diff
@@ -546,7 +803,9 @@ export class LiveSocket {
     }, afterMs)
   }
 
-  getHookCallbacks(hookName){ return this.hooks[hookName] }
+  getHookCallbacks(name){
+    return name && name.startsWith("Phoenix.") ? Hooks[name.split(".")[1]] : this.hooks[name]
```

Comment:
Instead of adding this logic here, would it be simpler to initialize the `hooks` property with all built-in hooks?